pkgs/by-name/ts/tsx/remove-docs.patch

```diff
@@ -0,0 +1,8 @@
+diff --git a/pnpm-workspace.yaml b/pnpm-workspace.yaml
+deleted file mode 100644
+index 28756fa..0000000
+--- a/pnpm-workspace.yaml
++++ /dev/null
+@@ -1,2 +0,0 @@
+-packages:
+-  - docs
```

Comment:
could probably just remove with `substituteInPlace`

Reply:
don't know if theres a difference between an empty `pnmp-workspaces.yaml` vs a nonexistent file to `pnmp` but this works as-is so i'd prefer to keep it

Reply:
Oh you're removing it. Then you should just rm the file in the patch phase. Don't bother with the patch.

Reply:
done